tests/wallet.rs

```diff
@@ -688,3 +688,20 @@ fn inscribe_gif() {
     ),
   )
 }
+
+#[test]
+fn wallet_balance() {
+  let rpc_server = test_bitcoincore_rpc::spawn_with(Network::Regtest, "ord");
+
+  CommandBuilder::new("--regtest wallet balance")
+    .rpc_server(&rpc_server)
+    .expected_stdout("0\n")
+    .run();
+
+  let _ = &rpc_server.mine_blocks(1);
+
+  CommandBuilder::new("--regtest wallet balance")
+    .rpc_server(&rpc_server)
+    .expected_stdout(format!("5000000000\n"))
```

Comment:
Since this doesn't have any interpolations, format is unnecessary:

```suggestion
    .expected_stdout("5000000000\n")
```

There's a just recipe for running some of the checks that run on ci, try `just ci`, or look for the ci recipe in the justfile to find the commands to run. CI runs clippy (which would complain about this), checks formatting, and runs bin/forbid, which forbids certain strings from appearing in the codebase.

bin/forbid is actually quite useful. It forbids the string `todo`, so if you want to make sure you don't forget something before comitting, write `todo: I need to do blah blah blah` and CI will complain, so you don't need to worry about forgetting it.

Reply:
ah! I haven't used `just` before. I'll check that out so you don't have to keep commenting on things that the linter would check. thanks!

Reply:
No worries! [just](https://github.com/casey/just/) is kind of like make, but only for saving and running commands, and less weird.

Reply:
yeah, this is great. should be a lot easier to make sure im conforming to style, etc. for future PRs :)